### PR TITLE
fix: Group Plans string duplication & style adjustment

### PR DESCRIPTION
**Pull request by CuriousMagpie**
- removed duplicate instance of "Choose your payment method" since that text is now part of the `payment-buttons` component and updated `h4` color to match current design
- adjusted the styling of `updating-group` so that the purple box and the payment buttons box connect with each other and display as if they are a single item instead of two boxes mashed together

### CODE REVIEW

File: website/client/src/components/groups/groupPlan.vue [hunk 1]
```diff
@@ -398,6 +403,10 @@
   .payment-options {
     margin-bottom: 4em;
 
+    h4 {
+      color: #34313a;
```
Comment (SabreCat) [inline website/client/src/components/groups/groupPlan.vue:407]: We don't necessarily need to fix all instances in this PR, but if we're adding a new color style we should take the opportunity to import `colors.scss` and use the `$color-XXX` variables.

**Review by SabreCat — CHANGES_REQUESTED** [review]
One tiny suggestion, I can tidy it up if that works better!

**Comment by CuriousMagpie** [comment]
Yeah, for sure--I've been trying to do that! Must have missed this one in
my excitement to round some corners to `border-radius: 8px`.
--
Natalie Luhrs
my pronouns are she/they

File: website/client/src/components/groups/groupPlan.vue (hunk 1 above)
Comment (SabreCat) [reply website/client/src/components/groups/groupPlan.vue:407]: On second thought, there's enough random hex codes in this file that we should probably do it all at one go later... never mind XD

**Review by SabreCat — APPROVED** [review]

**Pull request merged**